v1/lists.yaml

```diff
@@ -346,6 +357,15 @@ paths:
           description: Continuation parameter from previous request
           type: string
           required: false
+        - name: sort
+          in: query
+          description: |
+            Sort order
+            - `name` (default): by page title, ascending;
+            - `updated`: by last modification date, descending.
```

Comment:
Maybe more a client question. I'm leaning towards `updated` should be the default. Where is the default behavior implemented? Do we have tests?

Reply:
Defaults are on implemented in the MediaWiki API. I should probably implement it in the proxy though, it's not nice to depend on the defaults of another API.

I have no opinion on default behavior, I'll change it to `updated`.

Tests are still in WIP: #907 
On the MW side, there are DB tests but no API tests. I'll get there eventually; the API layer has very little logic in it though.

Reply:
Sounds good. My opinion on the default behavior is not very strong. Before you change it it might be good to check with @dbrant and whoever works on the iOS portion of this, what they actually prefer.
The main reason I brought it up is because I think that `updated` probably makes for a faster sync experience but I haven't fully thought through if the reverse chronological order makes issues more likely or not.